config/hubs/schema.yaml

```diff
@@ -74,6 +75,38 @@ properties:
       - project
       - cluster
       - zone
+  aws:
+    type: object
+    additionalProperties: false
+    description: |
+      Configuration to connect to the cluster on AWS. Is used when
+      provider is set to `aws`
+    properties:
+      key:
+        type: string
+        description: |
+          Path to a `sops` encrypted JSON key that contains AWS credentials
+          with enough rights to get full access to the kubernetes cluster.
+      clusterType:
+        type: string
+        description: |
+          We currently support kops and eks based kubernetes cluster on AWS.
+        enum:
+          - kops
+          - eks
+      clusterName:
+        type: string
+        description: |
+          Name of the kubernetes cluster.
```

Comment:
I guess there is an EKS concept of a name, but is kops installing k8s on instances without AWS being aware of the k8s cluster? Then does it mean that this name have meaning to kops separate to the meaning it would have if used by eksctl?

Reply:
I would say they have a separate meaning as long as with kops, you are deploying the whole k8s thing whereas, with EKS, the control plane is under Amazon guard. I should probably be more descriptive here as well...

Reply:
Or get rid of the kubernetes word :wink: